tests/virtctl/create_vm.go

```diff
@@ -0,0 +1,172 @@
+package virtctl
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"strconv"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	v1 "kubevirt.io/api/core/v1"
+	apiinstancetype "kubevirt.io/api/instancetype"
+	instancetypev1alpha2 "kubevirt.io/api/instancetype/v1alpha2"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+	"sigs.k8s.io/yaml"
+
+	"kubevirt.io/kubevirt/pkg/virtctl/create"
+	"kubevirt.io/kubevirt/tests/clientcmd"
+	"kubevirt.io/kubevirt/tests/libstorage"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+const cloudInitUserData = `#cloud-config
+user: user
+password: password
+chpasswd: { expire: False }`
+
+var _ = Describe("[sig-compute][virtctl]create vm", func() {
+	var virtClient kubecli.KubevirtClient
+
+	BeforeEach(func() {
+		var err error
+		virtClient, err = kubecli.GetKubevirtClient()
+		Expect(err).ToNot(HaveOccurred())
+	})
+
+	Describe("should create a valid VM manifest", func() {
+		It("VM with random name", func() {
+			out, err := runCmd()
+			Expect(err).ToNot(HaveOccurred())
+			_, err = virtClient.VirtualMachine(util.NamespaceTestDefault).Create(unmarshalVM(out))
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
Can you assert the name, that `running == false` etc? Anything we'd expect to be set by default.

Reply:
ACK